### PR TITLE
chore: review docs

### PR DESCRIPTION
**Pull request by filipesilva**
See https://github.com/angular/angular-cli/issues/1356

/cc @hansl @TheLarkInn

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -241,19 +233,47 @@ source ~/.bash_profile
...
+Simply install your library via `npm install lib-name` and import it in your code.
+
+If the library does not include typings, you can search and install them using `npm run typings` npm script provided:
+
+```bash
+npm install moment
+npm run typings -- search moment
+npm run typings -- install dt~moment --global
+```
+
```
Comment (TheLarkInn) [inline README.md]: Are we going to talk about @types and `npm install @types/foo`

Comment (filipesilva) [inline README.md]: We're not using @types in the webpack branch atm, and I actually tried now again and I can't seem to get it to work with either the build system or the ts-node for e2e tests. Have you had more success?

Comment (TheLarkInn) [inline README.md]: I haven't as of recent, we can add it to our nice-to-have-lets-get-it-after-the-branch-lands-in-master list

File: README.md [hunk 2]
```diff
@@ -241,19 +233,47 @@ source ~/.bash_profile
...
 
-We support all major CSS preprocessors:
+Angular-CLI supports all major CSS preprocessors:
 - sass (node-sass)
 - less (less)
 - compass (compass-importer + node-sass)
 - stylus (stylus)
 
-To use one just install for example `npm install node-sass` and rename `.css` files in your project to `.scss` or `.sass`. They will be compiled automatically.
+To use these prepocessors simply add the file to your component's `styreUrl`:
```
Comment (thiagogcm) [inline README.md]: Typo: preprocessors

Comment (filipesilva) [inline README.md]: Thanks for catching this, I'll try to include it in the next PR.

**Pull request merged**